### DISCUSSION TITLE
Android distribution uses JCenter

### DISCUSSION THREAD

**Discussion by malenalbc** [trustwallet/wallet-core#1363] [Q&A]
The `com.trustwallet:wallet-core` package [is being distributed via JCenter](https://mvnrepository.com/artifact/com.trustwallet/wallet-core), which will be [closing on May 1st](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/).

Are there any plans on distributing new releases through maven central?

**Comment by hewigovens** [comment]
checkout GitHub Packages https://github.com/trustwallet/wallet-core/packages/700258

**Comment by malenalbc** [comment]
Afaik Github packages needs authentication via a personal access token, which would force me to handle credentials on my `build.gradle` for just this dependency. What benefits have GP over Maven central?

**Comment by hewigovens** [comment]
It is so much easier for us to integrate: test, build and release all in GitHub Actions, I think you only need to create a token has minimum packages reading permission
